### PR TITLE
feat: add minikube completion spec

### PR DESCRIPTION
**Pull request by singhiqbal1007**
[Minikube](https://github.com/kubernetes/minikube) implements a local Kubernetes cluster on macOS, Linux, and Windows. 

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behavior? (You can also link to an open issue here)**
`minikube` commands are currently not-supported

**What is the new behavior (if this is a feature change)?**
Autocomplete and documentation to help guide developers using the `minikube` CLI

**Additional info:**
This PR resolves this issue #628

### CODE REVIEW

**Comment by singhiqbal1007** [comment]
@fedeci / @mschrage / @QuiiBz is there something that I am missing for this PR to get reviewed? I am new to open source I think I followed steps from the Guide. I would love to use minikube completions without dev mode 😛. Do let me know. Thnx.

File: src/minikube.ts [hunk 1]
```diff
@@ -0,0 +1,949 @@
...
+    name: ["--unset", "-u"],
+    description: "Unset variables instead of setting them",
+  },
+};
+
+const sharedArgs: Record<string, Fig.Arg> = {
+  flags: {
+    name: "[flags] [options]",
+  },
+};
```
Comment (fedeci) [inline src/minikube.ts:20]: Fig does not support this format for args ([see](https://fig.io/docs/getting-started/first-completion-spec#3-adding-args)).

File: src/minikube.ts [hunk 2]
```diff
@@ -0,0 +1,949 @@
+const sharedOpts: Record<string, Fig.Option> = {
+  output: {
+    name: ["--output", "-o"],
+    description: "Format to print stdout in. Options include: [text,json]",
```
Comment (fedeci) [inline src/minikube.ts:4]:
```suggestion
    description: "Format to print stdout in",
    args: {
      name: "format",
      suggestions: ["json", "text"],
    },
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thank you for the spec @singhiqbal1007, sorry for the late review😃

**Comment by singhiqbal1007** [comment]
@fedeci Thank you for taking out time for reviewing the PR. I have made changes as requested. I also added suggestions wherever I found necessary. Please review.

**Review by fedeci — APPROVED** [review]
Thanks @singhiqbal1007! I think there are some listing errors, you can run `npm run lint:fix` to automatically fix them, then we should be ready to merge this in!

**Comment by singhiqbal1007** [comment]
@fedeci I ran `lint:fix` command again today. There was no change in the file after running it. Plus I am using ESLint extension in vscode which automatically fix everything on saving the file. Let me know if i am missing something. Thnx

**Comment by fedeci** [comment]
In fact the linting action is not failing, thanks again!

**Pull request merged**